packages/utils/src/testUtilities/addCustomJestMatchers.js

```diff
@@ -25,13 +26,11 @@ class JestMatcherFactory {
     this.extendApi = extendApi;
   }
 
-  isAsync = config => config.matcher.constructor.name === 'AsyncFunction';
```

Comment:
This method was not working correctly for `async` class methods...

It seems there is not a 100% correct way to say if a method is async in code, so I used an `isAsync` option instead

Reply:
Nice approach! It happened to me a few times before.. I can feel you.